domain/src/main/java/com/anytypeio/anytype/domain/dataview/interactor/CreateDataViewObject.kt

```diff
@@ -168,7 +171,8 @@ class CreateDataViewObject(
 
     data class Result(
         val objectId : Id,
-        val objectType: Id?
+        val objectType: Id?,
+        val struct: Struct?
```

Comment:
do we need nullable here? we could as well return an empty map

Reply:
changed to empty map